From 952ab8f36e3c52a7ac6830ffd5b5c65f71fa0931 Mon Sep 17 00:00:00 2001
From: Aleix Pol <aleixpol@kde.org>
Date: Wed, 25 Oct 2017 13:20:37 +0200
Subject: Be flexible to systems without a gtkrc file in /etc

Always load both files and make the best out of them

BUG: 382291
---
 src/appearancegtk2.cpp | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/appearancegtk2.cpp b/src/appearancegtk2.cpp
index 92cbee3..1fd3ea2 100644
--- a/src/appearancegtk2.cpp
+++ b/src/appearancegtk2.cpp
@@ -188,7 +188,9 @@ bool AppearanceGTK2::loadSettings()
 {
     reset();
 
-    return loadSettingsPrivate("/etc/gtk-2.0/gtkrc") && loadSettingsPrivate(defaultConfigFile());
+    bool b = loadSettingsPrivate("/etc/gtk-2.0/gtkrc");
+    b |= loadSettingsPrivate(defaultConfigFile());
+    return b;
 }
 
 bool AppearanceGTK2::saveSettings() const
-- 
cgit v0.11.2

